shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RouteCardData.kt

```diff
@@ -254,7 +256,7 @@ data class RouteCardData(
          * [filterAtTime] and [filterAtTime] + [hideNonTypicalPatternsBeyondNext] are omitted.
          * Cancelled trips are also omitted when [context] = NearbyTransit.
          */
-        fun routeCardsForStopList(
+        suspend fun routeCardsForStopList(
```

Comment:
Per the PR checklist, since this will likely be expensive (although hopefully not very expensive), it belongs in a background thread.